ci/test_wheel.sh

```diff
@@ -9,9 +9,6 @@ RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen "${RAPIDS_CUDA_VERSION}")"
 CPP_WHEELHOUSE=$(RAPIDS_PY_WHEEL_NAME="librapidsmpf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-github cpp)
 PYTHON_WHEELHOUSE=$(RAPIDS_PY_WHEEL_NAME="rapidsmpf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-github python)
 
-# generate test dependencies
-rapids-dependency-file-generator --output requirements --file-key "test_python" --matrix "cuda=12.*;arch=x86_64;cuda_suffixed=true;py=313" | tee -a "$PIP_CONSTRAINT"
```

Comment:
I first looked at this line because the addition of `dask-cuda[cu12]` tripped `pip` up:

> _ERROR: Constraints cannot have extras_

([build link](https://github.com/rapidsai/rapidsmpf/actions/runs/17302519440/job/49118037236?pr=466#step:11:55))

On closer inspection, I think this line is broken and also unnecessary:

1. nothing in `dependencies.yaml` would match `py=313` (the `py:` entries are spelled with dots like `py: "3.13"`)
2. everything in `test_python` already ends up in the `[test]` extra of the wheels

Just removing it simplifies this script and avoids the need to update those hard-coded CUDA version, CPU architecture, and Python version as the RAPIDS support matrix changes.